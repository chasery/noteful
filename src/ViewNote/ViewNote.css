.NoteContainer {
  flex: 1;
}

.ControlContainer {
  margin-right: 1rem;
}
