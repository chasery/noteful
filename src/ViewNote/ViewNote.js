import React, { Component, Fragment } from "react";
import STORE from "../store";
import FolderList from "../FolderList/FolderList";
import NoteList from "../NoteList/NoteList";
import Note from "../Note/Note";
import "./ViewNote.css";

class ViewNote extends Component {
  render() {
    const note = STORE.notes.find(
      (note) => note.id === this.props.match.params.noteId
    );
    return (
      <Fragment>
        <FolderList></FolderList>
        <NoteList>
          <Note
            key={note.id}
            id={note.id}
            name={note.name}
            modified={note.modified}
            folderId={note.folderId}
            content={note.content}
          />
        </NoteList>
      </Fragment>
    );
  }
}

export default ViewNote;
