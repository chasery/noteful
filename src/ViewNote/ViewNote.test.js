import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import ViewNote from "./ViewNote";

describe("ViewNote component", () => {
  it("renders without error", () => {
    const wrapper = shallow(<ViewNote />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
