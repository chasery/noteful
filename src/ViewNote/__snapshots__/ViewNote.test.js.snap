// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewNote component renders without error 1`] = `
<div>
  ViewFolder
</div>
`;
